assets/lib/dialog/dialog.js

```diff
@@ -880,7 +880,7 @@
 			}
 
 			if (this.focusedButton) {
-				this.focusedButton.focus();
+				this.focusedButton.trigger('focus');
```

Comment:
shouldn't be fixed in https://github.com/kobizz/dialogs-manager?

Reply:
Correct. See: https://github.com/kobizz/dialogs-manager/pull/20

When the other library will be update I'll merge it to Elementor.

Meanwhile, @mark-elementor please remove the changes from the `dialog.js` file.

Reply:
not relevant any more, the `dialog.js` file was already updated.